ts/features/profile/saga/networking/handleGetProfile.ts

```diff
@@ -0,0 +1,46 @@
+import { ActionType } from "typesafe-actions";
+import { call, put } from "typed-redux-saga/macro";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { BackendClient } from "../../../../api/backend";
+import { SagaCallReturnType } from "../../../../types/utils";
+import { getProfile as loadProfile } from "../../store/actions";
+import { getGenericError, getNetworkError } from "../../../../utils/errors";
+
+/**
+ * Handle the remote call to retrieve the profile data
+ * @param getProfile
+ */
+export function* handleGetProfile(
+  getProfile: ReturnType<typeof BackendClient>["getProfile"],
+  _: ActionType<typeof loadProfile.request>
+) {
+  try {
+    const getProfileResult: SagaCallReturnType<typeof getProfile> = yield* call(
+      getProfile,
+      {}
+    );
+
+    if (getProfileResult.isRight()) {
+      // right side of the Either
+      if (getProfileResult.value.status === 200) {
+        yield* put(loadProfile.success(getProfileResult.value.value));
+        return;
+      } else {
+        yield* put(
+          loadProfile.failure({ ...getGenericError(new Error(`Error`)) })
+        );
+      }
+    }
+
+    if (getProfileResult.isLeft()) {
+      // left side of the Either
+      yield* put(
+        loadProfile.failure({
+          ...getGenericError(new Error(readableReport(getProfileResult.value)))
+        })
+      );
+    }
```

Comment:
What do you think about refactor in this way? 
```suggestion
      if (getProfileResult.value.status === 200) {
        yield* put(loadProfile.success(getProfileResult.value.value));
        return;
      } 
        yield* put(
          loadProfile.failure({ ...getGenericError(new Error(`Error`)) })
        );
        return:
    }
      yield* put(
        loadProfile.failure({
          ...getGenericError(new Error(readableReport(getProfileResult.value)))
        })
      );
    
```

Reply:
Sure is more compact. Anyway It's my personal opinion that sometimes using a "full if/then/else" makes the code more readable and less cognitive effort.

Reply:
Yes sure, feel free to ignore the comment.
Btw is the style used in the codebase till now.

Reply:
😄  The codebase style has the priority on my preferences...